src/components/dialog/dialog.scss

```diff
@@ -45,13 +45,18 @@
     }
     .mdc-dialog__container {
         height: 100%;
+        width: var(--dialog-width, auto);
     }
 
     .mdc-dialog__surface {
         width: var(--dialog-width, auto);
         height: var(--dialog-height, auto);
         background-color: var(--dialog-background-color);
         box-shadow: var(--shadow-depth-64);
+
+        max-width: var(--dialog-max-width, calc(100% - 32px));
+        max-height: var(--dialog-max-height, calc(100% - 32px));
+        border-radius: var(--dialog-border-radius, 4px);
```

Comment:
`px` 😱

Reply:
I copied it from the Material styles :upside_down_face:  You are more than welcome to improve it :smile: 

Is it possible to set it to something else as a default? I wanted it to use the value from Material if it has not been set but did not get it to work.

Reply:
I just meant we should use `rem` not `px` 😄 added a suggestion 😉